.travis.yml

```diff
@@ -37,6 +37,10 @@ matrix:
   - rust: nightly
 
 script:
+  - if [ $TRAVIS_RUST_VERSION = 'nightly' ]; then
```

Comment:
Maybe we can use the include feature: https://docs.travis-ci.com/user/build-matrix/#excluding-jobs

e.g.

```yaml
include:
- rust: nightly
- rust: nightly
  env: FEATURES="nightly protobuf push process"
- rust: stable
```

Reply:
Ah, yeah, that looks cleaner, didn't know about that.